airflow/providers/cohere/hooks/cohere.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         self.max_retries = max_retries
 
     @cached_property
-    def get_conn(self) -> cohere.Client:
+    def get_conn(self) -> cohere.Client:  # type: ignore[override]
```

Comment:
That not really related to the PR, but issue in general. Changes in cohere client somehow make `mypy` realise that this breaks types, in BaseHook it is expected to be a method, but here it defined as property.
We should change in the next major version